components/todo-input.js

```diff
@@ -1,20 +1,72 @@
-import React from "react";
+import React, { useState } from "react";
 import styled from "styled-components";
 
-export default function TodoInput() {
-  return (
-    <Wrapper>
-      리액트-투두
-      <br />
-      시간을 입력하는 칸
-      <br />
-      Todo를 입력하는 칸
-    </Wrapper>
-  );
+export default function TodoInput({ addTodo }) {
+    const [thisTodo, setThisTodo] = useState({
+    })
```

Comment:
```suggestion
    const [form, setForm] = useState({})
```
1. thisTodo는 `현재 입력중인 todo`라는 역할을 잘 설명하지 못 하는 것 같아요!
2. 빈 객체라서 줄바꿈 하지 않아도 될 것 같습니닿